package.json

```diff
@@ -4,22 +4,25 @@
   "version": "1.0.0",
   "license": "UNLICENSED",
   "scripts": {
-    "deps": "./tools/bin/run-create-relationships",
-    "deploy": "lifeomic-manual-deploy -p $npm_package_name -t lifeomic-test --action apply --skip-terraform"
+    "deps": "./tools/bin/run-create-relationships"
```

Comment:
`yarn deps` was surprising to me. I think `yarn create-relationships` is more obvious.